src/app/providers/category-whitelist/category-whitelist.ts

```diff
@@ -2,7 +2,9 @@ import { TestCategory } from '@dvsa/mes-test-schema/category-definitions/common/
 import { Injectable } from '@angular/core';
 
 // list all test categories that are able to be started by the users
-export const whiteList : TestCategory[] = [];
+export const whiteList : TestCategory[] = [
+  TestCategory.ADI2,
```

Comment:
do we want to check this in? if we do we need to remove it before we can release team journals

Reply:
Checked in so you can run it to check functionality if there was a desire. We can raise a ticket to remove categories from whitelist prior to Journal/Test Centre Journal release